File: app/src/main/java/com/example/igreeldownloader/models/InstaVideo.java
package com.example.igreeldownloader.models;

import java.util.ArrayList;

public class InstaVideo {
    public boolean error;
    public String msg;
    public ArrayList<Data> data;

    public class Data {
        public String url;
        public String format;
        public String ext;
        public String format_id;
    }

}

File: app/src/main/java/com/bluelock/realdownloader/models/InstaVideo.kt
package com.bluelock.realdownloader.models

class InstaVideo {
    var error = false
    var msg: String? = null
    var data: ArrayList<Data>? = null

    inner class Data {
        var url: String? = null
        var format: String? = null
        var ext: String? = null
        var format_id: String? = null
    }
}